### ISSUE TITLE
[BUG]: does "dotnet run" work on azure pipeline?

### ISSUE THREAD

**Issue by nguyenhmtriet** [microsoft/azure-pipelines-tasks#19677]
### New issue checklist

- [X] I searched for [existing GitHub issues](https://github.com/microsoft/azure-pipelines-tasks/issues)
- [X] I read [pipeline troubleshooting guide](https://docs.microsoft.com/vsts/build-release/actions/troubleshooting)
- [X] I checked [how to collect logs](https://learn.microsoft.com/azure/devops/pipelines/troubleshooting/review-logs?view=azure-devops)

### Task name

DotNetCoreCLI@2

### Task version

2.236.1

### Issue Description

I'm working on a migration console project that needs to run only once after deploying to Azure App Services.

I've configured the command `dotnet run --project src/MyProject.Migrator/MyProject.Migrator.csproj -c Release` on both `windows-2022` and `ubuntu-22.04` VM images. I've tried using both the `Powershell@2` and `DotNetCoreCLI@2` tasks, but unfortunately, neither of them seems to be effective.

It ends up with the same error the following below:
```
##[error]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
##[debug]Processed: ##vso[task.issue type=error;]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
```

![image](https://github.com/microsoft/azure-pipelines-tasks/assets/24971905/10d86c03-b2a5-4387-b47f-adf656fce50b)

### Environment type (Please select at least one enviroment where you face this issue)

- [ ] Self-Hosted
- [X] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

windows-2022 & ubuntu-22.04

### Relevant log output

```shell
D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(407,115): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(408,101): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
##[debug]Agent environment resources - Disk: D:\ Available 11292.00 MB out of 14333.00 MB, Memory: Used 2859.00 MB out of 7167.00 MB, CPU: Usage 4.05%
D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Application.Contracts\Shared\LookupDto.cs(5,21): warning CS8618: Non-nullable property 'Id' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Application.Contracts\LoanApplicationModule.Application.Contracts.csproj::TargetFramework=net8.0]
##[debug]Agent environment resources - Disk: D:\ Available 11232.00 MB out of 14333.00 MB, Memory: Used 3012.00 MB out of 7167.00 MB, CPU: Usage 3.94%

##[debug]Exit code 4294967254 received from tool 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
##[debug]STDIO streams have closed for tool 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
##[error]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
##[debug]Processed: ##vso[task.issue type=error;]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
##[warning].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
##[debug]Processed: ##vso[task.issue type=warning;].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
Info: Azure Pipelines hosted agents have been updated and now contain .Net 5.x SDK/Runtime along with the older .Net Core version which are currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. You can learn more about the breaking changes here: https://docs.microsoft.com/en-us/dotnet/core/tools/ and https://docs.microsoft.com/en-us/dotnet/core/compatibility/ . To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
##[debug]task result: Failed
##[error]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
##[debug]Processed: ##vso[task.issue type=error;]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
##[debug]Processed: ##vso[task.complete result=Failed;]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
Finishing: Dotnet run MyProject.Migrator
```

### Full task logs with system.debug enabled

<details>
<pre>
2024-03-22T16:19:46.7384215Z ##[debug]Evaluating condition for step: 'Dotnet run MyProject.Migrator'
2024-03-22T16:19:46.7385248Z ##[debug]Evaluating: SucceededNode()
2024-03-22T16:19:46.7385508Z ##[debug]Evaluating SucceededNode:
2024-03-22T16:19:46.7386003Z ##[debug]=> True
2024-03-22T16:19:46.7386338Z ##[debug]Result: True
2024-03-22T16:19:46.7386640Z ##[section]Starting: Dotnet run MyProject.Migrator
2024-03-22T16:19:46.7507798Z ==============================================================================
2024-03-22T16:19:46.7507956Z Task         : .NET Core
2024-03-22T16:19:46.7508049Z Description  : Build, test, package, or publish a dotnet application, or run a custom dotnet command
2024-03-22T16:19:46.7508182Z Version      : 2.236.1
2024-03-22T16:19:46.7508246Z Author       : Microsoft Corporation
2024-03-22T16:19:46.7508338Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli
2024-03-22T16:19:46.7508480Z ==============================================================================
2024-03-22T16:19:46.8147273Z ##[debug]Using node path: C:\agents\3.236.1\externals\node20_1\bin\node.exe
2024-03-22T16:19:46.9338187Z ##[debug]agent.TempDirectory=D:\a\_temp
2024-03-22T16:19:46.9346679Z ##[debug]loading inputs and endpoints
2024-03-22T16:19:46.9360978Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
2024-03-22T16:19:46.9381611Z ##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
2024-03-22T16:19:46.9384040Z ##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
2024-03-22T16:19:46.9387918Z ##[debug]loading INPUT_ARGUMENTS
2024-03-22T16:19:46.9390662Z ##[debug]loading INPUT_COMMAND
2024-03-22T16:19:46.9393534Z ##[debug]loading INPUT_CONFIGURATIONTOPACK
2024-03-22T16:19:46.9396690Z ##[debug]loading INPUT_INCLUDENUGETORG
2024-03-22T16:19:46.9400628Z ##[debug]loading INPUT_INCLUDESOURCE
2024-03-22T16:19:46.9403514Z ##[debug]loading INPUT_INCLUDESYMBOLS
2024-03-22T16:19:46.9409257Z ##[debug]loading INPUT_MODIFYOUTPUTPATH
2024-03-22T16:19:46.9412531Z ##[debug]loading INPUT_NOBUILD
2024-03-22T16:19:46.9417054Z ##[debug]loading INPUT_NOCACHE
2024-03-22T16:19:46.9420752Z ##[debug]loading INPUT_NUGETCONFIGPATH
2024-03-22T16:19:46.9424307Z ##[debug]loading INPUT_NUGETFEEDTYPE
2024-03-22T16:19:46.9427882Z ##[debug]loading INPUT_OUTPUTDIR
2024-03-22T16:19:46.9430541Z ##[debug]loading INPUT_PROJECTS
2024-03-22T16:19:46.9433885Z ##[debug]loading INPUT_PUBLISHPACKAGEMETADATA
2024-03-22T16:19:46.9436462Z ##[debug]loading INPUT_PUBLISHTESTRESULTS
2024-03-22T16:19:46.9438756Z ##[debug]loading INPUT_PUBLISHWEBPROJECTS
2024-03-22T16:19:46.9441014Z ##[debug]loading INPUT_REQUESTEDMAJORVERSION
2024-03-22T16:19:46.9444675Z ##[debug]loading INPUT_REQUESTEDMINORVERSION
2024-03-22T16:19:46.9447755Z ##[debug]loading INPUT_REQUESTEDPATCHVERSION
2024-03-22T16:19:46.9450080Z ##[debug]loading INPUT_SEARCHPATTERNPACK
2024-03-22T16:19:46.9452405Z ##[debug]loading INPUT_SEARCHPATTERNPUSH
2024-03-22T16:19:46.9454498Z ##[debug]loading INPUT_SELECTORCONFIG
2024-03-22T16:19:46.9456682Z ##[debug]loading INPUT_VERBOSITYPACK
2024-03-22T16:19:46.9458728Z ##[debug]loading INPUT_VERBOSITYRESTORE
2024-03-22T16:19:46.9460951Z ##[debug]loading INPUT_VERSIONINGSCHEME
2024-03-22T16:19:46.9463748Z ##[debug]loading INPUT_WORKINGDIRECTORY
2024-03-22T16:19:46.9466055Z ##[debug]loading INPUT_ZIPAFTERPUBLISH
2024-03-22T16:19:46.9473086Z ##[debug]loading SECRET_SQLSERVERSERVICEPRINCIPALCONNECTIONSTRING
2024-03-22T16:19:46.9475927Z ##[debug]loading SECRET_SYSTEM_ACCESSTOKEN
2024-03-22T16:19:46.9488893Z ##[debug]loaded 32
2024-03-22T16:19:46.9496291Z ##[debug]Agent.ProxyUrl=undefined
2024-03-22T16:19:46.9496982Z ##[debug]Agent.CAInfo=undefined
2024-03-22T16:19:46.9497489Z ##[debug]Agent.ClientCert=undefined
2024-03-22T16:19:46.9498029Z ##[debug]Agent.SkipCertValidation=undefined
2024-03-22T16:19:47.1748807Z ##[debug]command=run
2024-03-22T16:19:47.1749750Z ##[debug]projects=src/MyProject.Migrator/MyProject.Migrator.csproj
2024-03-22T16:19:47.1751756Z ##[debug]arguments=-c Release
2024-03-22T16:19:47.1754258Z ##[debug]publishWebProjects=true
2024-03-22T16:19:47.1755423Z ##[debug]zipAfterPublish=true
2024-03-22T16:19:47.1757318Z ##[debug]workingDirectory=D:\a\1\s
2024-03-22T16:19:47.1760740Z ##[debug]check path : D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.236.1\node_modules\azure-pipelines-tasks-packaging-common\module.json
2024-03-22T16:19:47.1764769Z ##[debug]adding resource file: D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.236.1\node_modules\azure-pipelines-tasks-packaging-common\module.json
2024-03-22T16:19:47.1765345Z ##[debug]system.culture=en-US
2024-03-22T16:19:47.1776736Z ##[debug]check path : D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.236.1\task.json
2024-03-22T16:19:47.1777321Z ##[debug]adding resource file: D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.236.1\task.json
2024-03-22T16:19:47.1777883Z ##[debug]system.culture=en-US
2024-03-22T16:19:47.1805992Z ##[debug]which 'C:\Windows\system32\chcp.com'
2024-03-22T16:19:47.1810555Z ##[debug]found: 'C:\Windows\system32\chcp.com'
2024-03-22T16:19:47.1812431Z ##[debug]C:\Windows\system32\chcp.com arg: ["65001"]
2024-03-22T16:19:47.1812994Z ##[debug]C:\Windows\system32\chcp.com arg: ["65001"]
2024-03-22T16:19:47.1813531Z ##[debug]exec tool: C:\Windows\system32\chcp.com
2024-03-22T16:19:47.1814046Z ##[debug]exec tool: C:\Windows\system32\chcp.com
2024-03-22T16:19:47.1814485Z ##[debug]arguments:
2024-03-22T16:19:47.1814892Z ##[debug]arguments:
2024-03-22T16:19:47.1815303Z ##[debug]   65001
2024-03-22T16:19:47.1815697Z ##[debug]   65001
2024-03-22T16:19:47.1818664Z [command]C:\Windows\system32\chcp.com 65001
2024-03-22T16:19:47.1925019Z Active code page: 65001
2024-03-22T16:19:47.1927166Z ##[debug]which 'dotnet'
2024-03-22T16:19:47.1936107Z ##[debug]found: 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
2024-03-22T16:19:47.1952992Z Info: .NET Core SDK/runtime 2.2 and 3.0 are now End of Life(EOL) and have been removed from all hosted agents. If you're using these SDK/runtimes on hosted agents, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version.
2024-03-22T16:19:47.1958589Z ##[debug]System.DefaultWorkingDirectory=D:\a\1\s
2024-03-22T16:19:47.1961751Z ##[debug]defaultRoot: 'D:\a\1\s'
2024-03-22T16:19:47.1962371Z ##[debug]findOptions.allowBrokenSymbolicLinks: 'false'
2024-03-22T16:19:47.1962886Z ##[debug]findOptions.followSpecifiedSymbolicLink: 'true'
2024-03-22T16:19:47.1963451Z ##[debug]findOptions.followSymbolicLinks: 'true'
2024-03-22T16:19:47.1963939Z ##[debug]findOptions.skipMissingFiles: 'false'
2024-03-22T16:19:47.1964460Z ##[debug]matchOptions.debug: 'false'
2024-03-22T16:19:47.1964898Z ##[debug]matchOptions.nobrace: 'true'
2024-03-22T16:19:47.1965364Z ##[debug]matchOptions.noglobstar: 'false'
2024-03-22T16:19:47.1965795Z ##[debug]matchOptions.dot: 'true'
2024-03-22T16:19:47.1966240Z ##[debug]matchOptions.noext: 'false'
2024-03-22T16:19:47.1966669Z ##[debug]matchOptions.nocase: 'true'
2024-03-22T16:19:47.1967123Z ##[debug]matchOptions.nonull: 'false'
2024-03-22T16:19:47.1967560Z ##[debug]matchOptions.matchBase: 'false'
2024-03-22T16:19:47.1968009Z ##[debug]matchOptions.nocomment: 'false'
2024-03-22T16:19:47.1968485Z ##[debug]matchOptions.nonegate: 'false'
2024-03-22T16:19:47.1968939Z ##[debug]matchOptions.flipNegate: 'false'
2024-03-22T16:19:47.1969434Z ##[debug]pattern: 'src/MyProject.Migrator/MyProject.Migrator.csproj'
2024-03-22T16:19:47.1978942Z ##[debug]findPath: 'D:\a\1\s\src\MyProject.Migrator\MyProject.Migrator.csproj'
2024-03-22T16:19:47.1979429Z ##[debug]statOnly: 'true'
2024-03-22T16:19:47.1979899Z ##[debug]found 1 paths
2024-03-22T16:19:47.1980357Z ##[debug]applying include pattern
2024-03-22T16:19:47.1980850Z ##[debug]adjustedPattern: 'D:\a\1\s\src/MyProject.Migrator/MyProject.Migrator.csproj'
2024-03-22T16:19:47.1986383Z ##[debug]1 matches
2024-03-22T16:19:47.1986817Z ##[debug]1 final results
2024-03-22T16:19:47.1988105Z ##[debug]which 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
2024-03-22T16:19:47.1988652Z ##[debug]found: 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
2024-03-22T16:19:47.1989143Z ##[debug]C:\hostedtoolcache\windows\dotnet\dotnet.exe arg: run
2024-03-22T16:19:47.1989657Z ##[debug]C:\hostedtoolcache\windows\dotnet\dotnet.exe arg: --project
2024-03-22T16:19:47.1990207Z ##[debug]C:\hostedtoolcache\windows\dotnet\dotnet.exe arg: D:\a\1\s\src\MyProject.Migrator\MyProject.Migrator.csproj
2024-03-22T16:19:47.1990742Z ##[debug]C:\hostedtoolcache\windows\dotnet\dotnet.exe arg: -c Release
2024-03-22T16:19:47.1993173Z ##[debug]exec tool: C:\hostedtoolcache\windows\dotnet\dotnet.exe
2024-03-22T16:19:47.1993640Z ##[debug]arguments:
2024-03-22T16:19:47.1994058Z ##[debug]   run
2024-03-22T16:19:47.1994494Z ##[debug]   --project
2024-03-22T16:19:47.1994952Z ##[debug]   D:\a\1\s\src\MyProject.Migrator\MyProject.Migrator.csproj
2024-03-22T16:19:47.1995411Z ##[debug]   -c
2024-03-22T16:19:47.1995835Z ##[debug]   Release
2024-03-22T16:19:47.1996552Z [command]C:\hostedtoolcache\windows\dotnet\dotnet.exe run --project D:\a\1\s\src\MyProject.Migrator\MyProject.Migrator.csproj -c Release
2024-03-22T16:19:48.0613660Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2531.00 MB out of 7167.00 MB, CPU: Usage 7.08%
2024-03-22T16:19:53.5066478Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2547.00 MB out of 7167.00 MB, CPU: Usage 6.71%
2024-03-22T16:19:58.7045984Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2379.00 MB out of 7167.00 MB, CPU: Usage 6.41%
2024-03-22T16:20:03.7598394Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2406.00 MB out of 7167.00 MB, CPU: Usage 6.12%
2024-03-22T16:20:08.8084855Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2391.00 MB out of 7167.00 MB, CPU: Usage 5.87%
2024-03-22T16:20:13.8648829Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2383.00 MB out of 7167.00 MB, CPU: Usage 5.64%
2024-03-22T16:20:18.9301023Z ##[debug]Agent environment resources - Disk: D:\ Available 11299.00 MB out of 14333.00 MB, Memory: Used 2369.00 MB out of 7167.00 MB, CPU: Usage 5.42%
2024-03-22T16:20:24.0916877Z ##[debug]Agent environment resources - Disk: D:\ Available 11298.00 MB out of 14333.00 MB, Memory: Used 2281.00 MB out of 7167.00 MB, CPU: Usage 5.21%
2024-03-22T16:20:29.1664799Z ##[debug]Agent environment resources - Disk: D:\ Available 11298.00 MB out of 14333.00 MB, Memory: Used 2450.00 MB out of 7167.00 MB, CPU: Usage 5.04%
2024-03-22T16:20:34.2258679Z ##[debug]Agent environment resources - Disk: D:\ Available 11298.00 MB out of 14333.00 MB, Memory: Used 2425.00 MB out of 7167.00 MB, CPU: Usage 4.86%
2024-03-22T16:20:39.2772285Z ##[debug]Agent environment resources - Disk: D:\ Available 11298.00 MB out of 14333.00 MB, Memory: Used 2413.00 MB out of 7167.00 MB, CPU: Usage 4.71%
2024-03-22T16:20:44.3332603Z ##[debug]Agent environment resources - Disk: D:\ Available 11298.00 MB out of 14333.00 MB, Memory: Used 2415.00 MB out of 7167.00 MB, CPU: Usage 4.57%
2024-03-22T16:20:49.3863710Z ##[debug]Agent environment resources - Disk: D:\ Available 11297.00 MB out of 14333.00 MB, Memory: Used 2497.00 MB out of 7167.00 MB, CPU: Usage 4.42%
2024-03-22T16:20:54.4316530Z ##[debug]Agent environment resources - Disk: D:\ Available 11296.00 MB out of 14333.00 MB, Memory: Used 2489.00 MB out of 7167.00 MB, CPU: Usage 4.30%
2024-03-22T16:20:59.5138355Z ##[debug]Agent environment resources - Disk: D:\ Available 11294.00 MB out of 14333.00 MB, Memory: Used 2633.00 MB out of 7167.00 MB, CPU: Usage 4.19%
2024-03-22T16:21:00.7947739Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.Extended.cs(2,7): warning CS0105: The using directive for 'LoanApplicationModule' appeared previously in this namespace [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7951788Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.Extended.cs(3,7): warning CS0105: The using directive for 'LoanApplicationModule' appeared previously in this namespace [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7953189Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplicationManager.Extended.cs(2,7): warning CS0105: The using directive for 'LoanApplicationModule' appeared previously in this namespace [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7954492Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplicationManager.Extended.cs(3,7): warning CS0105: The using directive for 'LoanApplicationModule' appeared previously in this namespace [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7955807Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'FirstName' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7957339Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'LastName' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7958655Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'SocialSecurityNumber' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7960741Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'EmailAddress' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7962113Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'Street' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7963394Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'City' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7964678Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'State' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7966213Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'Country' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7967552Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'EmployerOrBusinessName' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7968882Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'EmployerOrBusinessAddress' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7970194Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'EmployerOrBusinessPhone' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7971593Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'PositionTitle' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7974308Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'FinancialInstitution' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7975950Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'AccountNumber' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:00.7977701Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplications\LoanApplication.cs(192,19): warning CS8618: Non-nullable property 'BorrowerName' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Domain\LoanApplicationModule.Domain.csproj::TargetFramework=net8.0]
2024-03-22T16:21:01.7516275Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(319,95): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7525365Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(320,75): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7532905Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(320,97): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7540840Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(321,93): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7545952Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(322,71): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7551718Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(322,89): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7556374Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(323,79): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7561531Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(323,105): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7567766Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(325,117): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7572526Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(329,78): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7577415Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(329,103): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7581988Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(333,81): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7586442Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(333,109): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7590769Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(334,74): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7595284Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(334,95): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7599534Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(335,74): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7604237Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(335,95): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7608549Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(336,74): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7613042Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(336,95): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7617874Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(337,101): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7622648Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(338,89): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7627519Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(339,85): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7632110Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(340,87): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7636881Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(343,91): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7641557Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(351,121): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7646280Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(352,127): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7651390Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(353,123): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7656360Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(354,103): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7663832Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(374,117): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7669425Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(375,103): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7674395Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(382,76): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7679563Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(382,99): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7683848Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(383,89): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7688350Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(383,125): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7692568Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(400,95): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7697037Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(400,137): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7701330Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(401,76): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7705803Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(401,99): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7709967Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(402,98): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7714382Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(402,143): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7718584Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(403,83): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7724010Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(403,113): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7728087Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(404,86): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7732565Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(404,119): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7736759Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(405,93): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7741241Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(405,133): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7745546Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(406,84): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7749999Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(406,115): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7754114Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(407,84): warning CS8602: Dereference of a possibly null reference. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7758524Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(407,115): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:01.7760211Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplications\EfCoreLoanApplicationRepository.cs(408,101): warning CS8604: Possible null reference argument for parameter 'value' in 'bool string.Contains(string value)'. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.EntityFrameworkCore\LoanApplicationModule.EntityFrameworkCore.csproj]
2024-03-22T16:21:06.2521863Z ##[debug]Agent environment resources - Disk: D:\ Available 11292.00 MB out of 14333.00 MB, Memory: Used 2859.00 MB out of 7167.00 MB, CPU: Usage 4.05%
2024-03-22T16:21:07.0380780Z D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Application.Contracts\Shared\LookupDto.cs(5,21): warning CS8618: Non-nullable property 'Id' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [D:\a\1\s\modules\LoanApplicationModule\src\LoanApplicationModule.Application.Contracts\LoanApplicationModule.Application.Contracts.csproj::TargetFramework=net8.0]
2024-03-22T16:21:11.3730708Z ##[debug]Agent environment resources - Disk: D:\ Available 11232.00 MB out of 14333.00 MB, Memory: Used 3012.00 MB out of 7167.00 MB, CPU: Usage 3.94%
2024-03-22T16:21:11.9991393Z 
2024-03-22T16:21:12.0456226Z ##[debug]Exit code 4294967254 received from tool 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
2024-03-22T16:21:12.0485475Z ##[debug]STDIO streams have closed for tool 'C:\hostedtoolcache\windows\dotnet\dotnet.exe'
2024-03-22T16:21:12.0556860Z ##[error]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
2024-03-22T16:21:12.0564620Z ##[debug]Processed: ##vso[task.issue type=error;]Error: The process 'C:\hostedtoolcache\windows\dotnet\dotnet.exe' failed with exit code 4294967254
2024-03-22T16:21:12.0568503Z ##[warning].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
2024-03-22T16:21:12.0569897Z ##[debug]Processed: ##vso[task.issue type=warning;].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
2024-03-22T16:21:12.0587389Z Info: Azure Pipelines hosted agents have been updated and now contain .Net 5.x SDK/Runtime along with the older .Net Core version which are currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. You can learn more about the breaking changes here: https://docs.microsoft.com/en-us/dotnet/core/tools/ and https://docs.microsoft.com/en-us/dotnet/core/compatibility/ . To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
2024-03-22T16:21:12.0598048Z ##[debug]task result: Failed
2024-03-22T16:21:12.0602999Z ##[error]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
2024-03-22T16:21:12.0603625Z ##[debug]Processed: ##vso[task.issue type=error;]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
2024-03-22T16:21:12.0633886Z ##[debug]Processed: ##vso[task.complete result=Failed;]Dotnet command failed with non-zero exit code on the following projects : [
  'D:\\a\\1\\s\\src\\MyProject.Migrator\\MyProject.Migrator.csproj'
]
2024-03-22T16:21:12.0637830Z ##[section]Finishing: Dotnet run MyProject.Migrator
</pre>
</details>

### Repro steps

```yml
trigger:
  branches:
    include:
      - main
      - release/*

variables:
  buildConfiguration: Release


pool:
  name: Azure Pipelines
  vmImage: 'ubuntu-22.04'

stages:
  - stage: BuildWebApps
    displayName: Build Web Apps
    jobs:
    - job: Build
      steps:
      - checkout: self
        clean: true
    ...

  - stage: DeployWebApps
    displayName: Deploy Web Apps
    dependsOn: BuildWebApps
    ...

  - stage: Migration
    displayName: Run migration
    dependsOn: DeployWebApps
    pool:
      vmImage: 'windows-2022'
    jobs:
    - job: RunMigration
      steps:
        - checkout: self
          clean: true

        - task: AzureKeyVault@2
          inputs:
            azureSubscription: Deployment_ServiceConnection
            KeyVaultName: mortagepos-keyvault-test
            SecretsFilter: 'SqlServerServicePrincipalConnectionString'
            RunAsPreJob: false
            
        - task: AzureCLI@2
          inputs:
            azureSubscription: Deployment_ServiceConnection
            scriptType: 'pscore'
            scriptLocation: 'inlineScript'
            addSpnToEnvironment: true
            inlineScript: |
              az login --service-principal -u $env:servicePrincipalId -p $env:servicePrincipalKey --tenant $env:tenantId

        - task: UseDotNet@2
          displayName: Install .NET 8.0.x Sdk
          inputs:
            packageType: sdk
            version: 8.0.x

        - task: DotNetCoreCLI@2
          displayName: 'Dotnet run MyProject.Migrator'
          env:
            ConnectionStrings__Default: $(SqlServerServicePrincipalConnectionString)
            DOTNET_ENVIRONMENT: Production
          inputs:
            command: 'run'
            projects: 'src/MyProject.Migrator/MyProject.Migrator.csproj'
            arguments: '-c $(buildConfiguration)'
```

**Comment by KonstantinTyukalov** [comment]
Hi @nguyenhmtriet, I'm unable to reproduce your issue, so it's probably your project specific issue.
Could you try to separate restore and run steps, and enable diagnostic mode for both steps to see the output:

```yaml
  - task: DotNetCoreCLI@2
    inputs:
      command: 'restore'
      projects: 'src/MyProject.Migrator/MyProject.Migrator.csproj'
      feedsToUse: 'select'
      verbosityRestore: 'Diagnostic'

  - task: DotNetCoreCLI@2
    displayName: 'Dotnet run MyProject.Migrator'
    env:
      ConnectionStrings__Default: $(SqlServerServicePrincipalConnectionString)
      DOTNET_ENVIRONMENT: Production
    inputs:
      command: 'run'
      projects: 'src/MyProject.Migrator/MyProject.Migrator.csproj'
      arguments: '-c $(buildConfiguration) --verbosity diagnostic --no-restore'
   ```
